src/pages-and-resources/discussions/data/thunks.js

```diff
@@ -76,3 +88,9 @@ export function saveAppConfig(courseId, appId, drafts, successPath) {
     }
   };
 }
+
+export function updatedDiscussionTopics(payload) {
```

Comment:
Arguably this doesn't need to be a thunk, since it's not asynchronous and is just delegating to a single action.